### ISSUE TITLE
Barcode isn't well displayed in  PDF documents

### ISSUE THREAD

**Issue by alexisgstalter** [Holger-Will/ean13-font#1]
Good morning,

I'm trying to generate a PDF document with a barcode generated with your font. The problem is that the barcode is totally smashed inside  : http://prntscr.com/h1ql8p

but is rendering correctly in the webpage : http://prntscr.com/h1qliy

I used Code128 fonts to do similar things, and it worked, my PDF library use the CSS inline and external that is mentionned in the HTML code... but it doesn't work with your font.

**Comment by Holger-Will** [comment]
that looks like there are only the stop codes. Do you encode the text properly? can you post an encoded string please.

**Comment by alexisgstalter** [comment]
yeah sure, the encoded string is : _3*L4L6G6G6G7L3**R4R7R3R6R2R1*

(the ***** caract are also between _3 and L4, and at the end of the string, but it's corrected)

**Comment by Holger-Will** [comment]
how do you generate the PDF? One explanaion is that the software you use to render the PDF does not support ligatures... i have used LaTeX und postscript to generate PDF so far. both worked well...

**Comment by alexisgstalter** [comment]
I use IronPDF, I can write Code128 barcodes on it (with libre code 128 font for example)

**Comment by Holger-Will** [comment]
i can not test ironPDF (no Windows machine available) and the PDF's i created so far do work... It might be a font loading issue, but on the other hand the font does get loaded... my guess is still missing ligature support. Does the libre code 128 font use ligatures? My code 128 font does not...

**Comment by alexisgstalter** [comment]
It seems like it could be the problem, I don't know if libre font 128 (it's a google font) use ligatures, but I will assume that it doesn't. I tried with SelecPDF library as well, it doesn't work, I believe that it's a CSS problem (I use Bootstrap).

**Comment by Holger-Will** [comment]
I just realised that the web example (and the ean13_all.css just references the woff font without fallback. What you could try to do is install the ttf font and then use the installed font. try it with different tools. the installed font should work everywhere on windows (word, editor, paint...).
